Neos.Neos/Classes/Domain/Service/FusionService.php

```diff
@@ -305,4 +268,18 @@ public function setAppendFusionIncludes(array $appendFusionIncludes)
     {
         $this->appendFusionIncludes = $appendFusionIncludes;
     }
+
+    private function prepareFusionIncludes(array $fusionIncludes, string $filePathForRelativeResolves): FusionSourceCodeCollection
```

Comment:
please make this `protected` so other (monocle) packages may use it

Reply:
![image](https://user-images.githubusercontent.com/85400359/200027849-53448993-e2e8-4e25-8113-000afd7b7b7b.png)

Reply:
Having a custom FusionService that extends the original one is not evil and we have at least one valid use case for that.

Reply:
yes sorry i was not trying to be mean. Its just that our whole `FusionService` is broken.

My masterplan is to remove the whole `FusionService` at some point and so i was trying to actually limit the api surface.

Reply:
Actually it would be fine to declare the scope of those methods whatever you like if you add something like this:

```
public function getMergedFusionObjectTreeForSitePackage(FlowPackageInterface $sitePackage)
```

In which case monocle would not extend the service but simply use it.

Reply:
a little lobbyism never did any harm ^^

I really like this api and clear approach!

Reply:
after thinking A LOT and also consulting you @nezaniel and @kitsunet i found a clean api for every usecase:

```php
$fusionConfiguration = $this->fusionParser->parseFromSource(
    $this->fusionSourceCodeFactory->createFromNodeTypeDefinitions()
        ->union(
            $this->fusionSourceCodeFactory->createFromAutoIncludes()
        )
        ->union(
            FusionSourceCodeCollection::tryFromPackageFusionRoot("Vendor.Site")
        )
);
```

i hope you like it ;)